boards/x86/gpmrb/Kconfig.board

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config BOARD_GPMRB
+	bool "Gordon Peak MRB"
```

Comment:
Seems a bit weird that these symbols have a prompt. I wonder if going into the `menuconfig` and changing the board there was ever expected to work. Might not if other settings for the board are set by the `-D<BOARD>`.

Unrelated though, so could keep it for consistency. :)

Reply:
Yeah.. for the moment I'm just following suit, rather than trying to set a standard.